tests/integration/kubernetes/test_kubernetes_pod_operator.py

```diff
@@ -61,7 +72,10 @@ def setUp(self):
                 'namespace': 'default',
                 'name': ANY,
                 'annotations': {},
-                'labels': {'foo': 'bar'}
+                'labels': {'foo': 'bar',
+                           'exec_date': '2019-01-01T0900000000-4bf747daa',
```

Comment:
I wonder if it's worth using a non-standard date format here, rather than always hashing this label? (Ultimately it doesn't matter much to us as Airflow I guess.)